tests/buildtest/mod.rs

```diff
@@ -23,6 +23,28 @@ fn test_hello_world() {
     });
 }
 
+#[test]
+fn test_process_lines() {
+    runner::run("hello-world", |run| {
+        run.build(SandboxBuilder::new().enable_networking(false), |build| {
+            let storage = rustwide::logging::LogStorage::new(LevelFilter::Info);
+            let mut ex = false;
+            rustwide::logging::capture(&storage, || -> Result<_, Error> {
+                build
+                    .cargo()
+                    .process_lines(&mut |_line: &str| ex = true)
```

Comment:
This only checks that `process_lines` is called *eventually*, could you check that a known line outputted by `cargo run` was provided to it?